crypto/aes/asm/aes-s390x.pl

```diff
@@ -1405,7 +1405,42 @@ ()
 	clr	%r0,%r1
 	jl	.Lctr32_software
 
-	stm${g}	%r6,$s3,6*$SIZE_T($sp)
+	stm${g}	$s2,$s3,10*$SIZE_T($sp)
+
+	clr	$len,%r1
```

Comment:
Could you add "lack of 'g' is not a typo" comment to this line? Or "does work even in 64-bit mode"?

Reply:
Will do.

Reply:
BTW, can you share how much faster kma is?

Reply:
i sent you a mail in this regard.